src/api/app/models/user.rb

```diff
@@ -749,7 +739,14 @@ def owned_packages
 
   # lists reviews involving this user
   def involved_reviews(search = nil)
-    BsRequest.collection(user: login, roles: %w(reviewer creator), reviewstates: %w(new), states: %w(review), search: search).not_creator(login)
+    result = BsRequest.where(reviews: { reviewable_id: id, reviewable_type: 'User' }).or(
+      BsRequest.where(reviews: { reviewable_id: involved_projects_ids, reviewable_type: 'Project' }).or(
+        BsRequest.where(reviews: { reviewable_id: involved_packages_ids, reviewable_type: 'Package' }).or(
+          BsRequest.where(reviews: { reviewable_id: groups.pluck(:id), reviewable_type: 'Group' })
+        )
```

Comment:
I'm thinking, maybe we can use scopes to make this nicer? i.e.

bs_request.rb
```ruby
  scope :for_users, ->(user_ids) { where(reviewable_id: user_ids, reviewable_type: 'User') }
  scope :for_projects, ->(project_ids) { where(reviewable_id: project_ids, reviewable_type: 'Project') }
  scope :for_packages, ->(package_ids) { where(reviewable_id: package_ids, reviewable_type: 'Package') }
  scope :for_groups, ->(group_ids) { where(reviewable_id: group_ids, reviewable_type: 'Group') }
```

user.rb
```ruby
  def involved_reviews(search = nil)
    BsRequest.for_users([id]).or(
      BsRequest.for_projects(involved_projects_ids).or(
        BsRequest.for_packages(involved_packages_ids).or(
          BsRequest.for_groups(groups.pluck(:id))
        )
      )
    )
  end
```